source/deployment-considerations/colocation.txt

```diff
@@ -0,0 +1,105 @@
+.. _rm-colocation:
+
+==========
+Colocation
+==========
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+In the context of Relational Migrator, colocation refers to where the 
```

Comment:
As above, I don't think this term is helpful. "Colocation" could imply running multiple things on the same machine, which we don't want to encourage.